integration-tests/olminstall/helpers/oc_util.py

```
@@ -0,0 +1,57 @@
+
+from .errors import AppError
+
+
+def run_cmd(
+    cmd: list[str],
+    *,
+    capture: bool = True,
+    check: bool = True,
+    input_text: str | None = None,
+) -> subprocess.CompletedProcess[str]:
+    proc = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)